### ISSUE TITLE
implement `ondevice status`

### ISSUE THREAD

**Issue by mreithub** [ondevice/ondevice-client#16]
Should print the current device ID.

The output might look something like this:

```
Device is ONLINE
ID: someUser/someDev
```

The device ID should be print to stdout, everything else to stderr.
The exit code should indicate whether or not the device is online (0: 

Nice to have: `--json` flag (prints the complete status as JSON object - might be the RAW response the device sends through the unix socket)

Depends on #5.

**Comment by mreithub** [comment]
a quick-n-dirty version has been implemented in the meantime (only checking if the `ondevice daemon` is running, but having no idea if it's really online):

```
Device:
  ID: ondevice.abcdef
Client:
  version: 0.2.4
```

missing features (as of now):
- a machine readable version of this
- a simple way to get the `ondeviceId`
- actual information on the `ondevice daemon` status (online/offline, software version, device ID)
- maybe information on the currently opened tunnels

**Comment by mreithub** [comment]
pretty much implemented in af3bceb8
